controllers/Index.php

```diff
@@ -434,8 +434,20 @@ protected function addPageSyntaxFields($formWidget, $page)
                 unset($fieldConfig['fields']);
             }
 
-            $fieldConfig['cssClass'] = 'secondary-tab ' . array_get($fieldConfig, 'cssClass', '');
-            $formWidget->secondaryTabs['fields']['viewBag[' . $fieldCode . ']'] = $fieldConfig;
+			$placement = (!empty($fieldConfig['placement']) ? $fieldConfig['placement'] : NULL);
+
+			switch($placement) {
+
```

Comment:
Ok, I have fixed this.

_And this is my first PR so please let me know if simply add new commit is enough. Thanks :)_